packages/utils/telemetry-utils/src/test/childLogger.spec.ts

```diff
@@ -176,4 +177,62 @@ describe("ChildLogger", () => {
 		childLogger2.send({ category: "generic", eventName: "testEvent" });
 		assert(sent, "event should be sent");
 	});
+
+	describe("Sampling", () => {
+		let events: ITelemetryBaseEvent[] = [];
+		function getBaseLoggerWithConfig(
+			configDictionary?: Record<string, ConfigTypes>,
+		): ITelemetryBaseLogger {
+			const logger: ITelemetryBaseLogger = {
+				send(event: ITelemetryBaseEvent): void {
+					events.push(event);
+				},
+			};
+			const configProvider = (
+				settings: Record<string, ConfigTypes>,
+			): IConfigProviderBase => ({
+				getRawConfig: (name: string): ConfigTypes => settings[name],
+			});
+			return mixinMonitoringContext(logger, configProvider(configDictionary ?? {})).logger;
+		}
+
+		beforeEach(() => {
+			events = [];
+		});
+
+		it("Systematic Sampling works as expected", () => {
+			const injectedSettings = {
+				"Fluid.Telemetry.DisableSampling": true,
+			};
+			const logger = getBaseLoggerWithConfig(injectedSettings);
+			const childLogger = ChildLogger.create(logger);
```

Comment:
This is probably unnecessary, right? We can pass `logger` directly to `createSampledLoggerSend()`.

Reply:
Updates. Also these tests were moved to utils.spec.ts